packages/ignite-cli/src/commands/sourceMap.js

```diff
@@ -0,0 +1,75 @@
+// @cliDescription  Digests the source map file to display proper feedback
+// @cliAlias source-map
+// ----------------------------------------------------------------------------
+
+const SourceMapConsumer = require('source-map').SourceMapConsumer
+const exitCodes = require('../lib/exitCodes')
+
+const printSourceResult = (smc, position, colors) => {
+  const {source, line, column, name} = smc.originalPositionFor(position)
+  const styledName = colors.red(name)
+  const styledSource = colors.yellow(source)
+
+  console.log(`${styledName}: ${styledSource}
+  Line: ${line}
+  Column: ${column}
+  `)
+}
+
+const handleFile = (smc, stackData, colors) => {
+  // parse file for all line:column numbers
+  const regexString = /(\d+):(\d+)\)/g
+  let matchez = regexString.exec(stackData)
+
+  // Loop through matches and print
+  while (matchez) {
+    printSourceResult(smc, {
+      line: matchez[1],
+      column: matchez[2]
+    }, colors)
+    matchez = regexString.exec(stackData)
+  }
+}
+
+/*
+* Example 1:  To get one specific mapping
+* ignite source-map --map ./someFolder/index.ios.map --line 32 --column 76
+* ignite source-map -m ./someFolder/index.ios.map -l -32 -c 76
+*
+* Example 2:  To parse a stack trace string and convert it to usable output
+* ignite source-map --mapfile ./someFolder/index.ios.map --stackfile stackTrace.txt
+* ignite source-map -m ./someFolder/index.ios.map -s stackTrace.txt
+*/
+module.exports = async function (context) {
+  const { print, parameters, filesystem } = context
+  const { colors, error } = print
+  const { options } = parameters
+  // get options
+  const mapfile = options.mapfile || options.m
+  const line = options.line || options.l
+  const column = options.column || options.c
+  const stackfile = options.stackfile || options.s
+
+  // make sure mapfile is passed with line+column || stackfile supplied
+  const optionsValid = (mapfile && ((line && column) || stackfile))
+  if (!optionsValid) {
+    error(`You must pass a mapfile AND either a line+column or a stackfile.
+    e.g.
+    ignite source-map --map ./someFolder/index.ios.map --line 32 --column 76
```

Comment:
How would you feel about upgrading the required fields to parameters from options?

```sh
# a line
ignite source-map ./someFolder/index.ios.map 32

# a line + column
ignite source-map ./someFolder/index.ios.map 32:76

# a stackfile
ignite source-map ./someFolder/index.ios.map stackTrace.txt

# an error
ignite source-map ./someFolder/index.ios.map
```